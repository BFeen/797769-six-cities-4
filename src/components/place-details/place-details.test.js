import React from "react";
import renderer from "react-test-renderer";
import PlaceDetails from "./place-details.jsx";


const offers = [
  {
    id: 0,
    title: `Beautiful & luxurious apartment at great location`,
    picture: `img/apartment-01.jpg`,
    isPremium: true,
    price: 200,
    type: `Apartment`,
    rating: 4,
    coordinates: [
      52.3909553943508,
      4.85309666406198
    ],
  }, {
    id: 1,
    title: `Wood and stone place`,
    picture: `img/apartment-03.jpg`,
    isPremium: true,
    price: 170,
    type: `House`,
    rating: 5,
    coordinates: [
      52.369553943508,
      4.85309666406198
    ]
  }, {
    id: 2,
    title: `Canal view Princengracht`,
    picture: `img/room.jpg`,
    isPremium: false,
    price: 70,
    type: `Room`,
    rating: 3,
    coordinates: [
      52.3909553943508,
      4.929309666406198
    ]
  }, {
    id: 3,
    title: `Nice, cozy, warm big bed apartment`,
    picture: `img/apartment-02.jpg`,
    isPremium: false,
    price: 150,
    type: `Apartment`,
    rating: 4,
    coordinates: [
      52.3809553943508,
      4.939309666406198
    ]
  }
];

describe(`PlaceDetails snapshot test`, () => {
  it(`PlaceDetails rendering`, () => {
    const tree = renderer.create(
        <PlaceDetails
          mapClassName={`cities`}
          offerId={1}
          offers={offers}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
