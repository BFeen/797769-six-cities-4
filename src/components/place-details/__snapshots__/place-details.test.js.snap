// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceDetails snapshot test PlaceDetails rendering 1`] = `ShallowWrapper {}`;
